app/src/main/java/openfoodfacts/github/scrachx/openfood/views/ContinuousScanActivity.java

```diff
@@ -324,7 +329,8 @@ public void onError() {
                         fragmentTransaction.replace(R.id.frame_layout, productFragment);
                         fragmentTransaction.setTransition(FragmentTransaction.TRANSIT_FRAGMENT_FADE);
                         fragmentTransaction.commit();
-                        showFirstScanTooltipIfNeeded();
+
+
```

Comment:
You can reformat the code and remove these extra blank lines

Reply:
yep, will do